dss/operations/sync.py

```diff
@@ -26,6 +30,47 @@ def _log_warning(**kwargs):
     logger.warning(json.dumps(kwargs))
 
 
+@sync.action('verify-sync-all',
+             arguments={"--source-replica": {'choices': [r.name for r in Replica], 'required': True},
+                        "--destination-replica": {'choices': [r.name for r in Replica], 'required': True},
+                        "--since": {'required': False, 'help': "Only check objects newer than this (DSS_VERSION)"
+                                                               " e.g. 1970-01-01T000000.000000"}})
+def verify_sync_all(argv: typing.List[str], args: argparse.Namespace):
+    """
+    Like verify-sync, but provides keys from the designated source replica
+    (so that keys don't need to be supplied individually).
+    """
+    def list_objects_since(replica, since: datetime.datetime):
+        replica = Replica[replica]
+        handle = Config.get_blobstore_handle(replica)
+        for prefix in (BUNDLE_PREFIX, FILE_PREFIX, BLOB_PREFIX):
+            for name, metadata in handle.list_v2(replica.bucket, prefix=prefix):
```

Comment:
can you change this to `key, metadata` its what the other scripts use, easier for an operator to keep it in context.